doc/faq/howto_faq.rst

```diff
@@ -37,6 +37,38 @@ If you only want to use the `pandas` converter for `numpy.datetime64` values ::
 
 
 
+.. _howto-figure-empty:
+
+Check whether a figure is empty
+-------------------------------
```

Comment:
So, the next section uses `findobj`, can these two sections not be combined?  

I think it would be nice if this info also went into `figure.findobj`.  I was a little confused by this - I'd have just checked `figure.get_children()` `findobj` traverses the whole tree, I suspect needlessly for this case?

Reply:
Yes, `get_children()` is sufficient. Thanks, changed.

I don't know how to combine these sections - maybe don't have to anymore because they now use different methods. Will modify docstrings in a separate PR. Otherwise we can't port this back to the doc branch.